pkg/mutation/types/mutator.go

```diff
@@ -19,10 +20,16 @@ type ID struct {
 	Name      string
 }
 
+func (id ID) String() string {
+	return fmt.Sprintf("%v %v",
```

Comment:
Is this human-readable (the intent of string is to be human-readable), %s would call the "String()" method of these objects.

Reply:
%v uses the default formatting of objects. In this case, it calls the String() method as the objects satisfy the Stringer inferface. So this already does that.

Per: https://golang.org/pkg/fmt/

"Except when printed using the verbs %T and %p, special formatting considerations apply for operands that implement certain interfaces. In order of application:

...

5. If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any)."